### DISCUSSION TITLE
Proposal: Adjusting the height of background displaying in the song list.

### DISCUSSION THREAD

**Discussion by toiglak** [ppy/osu#12993] [Ideas]
I think that option should be implemented in future editor.
Default:
![wrong](https://user-images.githubusercontent.com/37531387/44094437-74bd708e-9fd6-11e8-9507-c9cfbf1f7dff.png)
Adjusted:
![correct](https://user-images.githubusercontent.com/37531387/44094442-77c994d8-9fd6-11e8-8b90-6c5d6be78bdd.png)

**Comment by peppy** [comment]
This is something which will be coming, but not until beatmap submission is moved across to lazer (which is to say, still some way off).
